small-talk/tailwind.config.js

```diff
@@ -1,10 +1,10 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    "./pages/**/*.{js,ts,jsx,tsx,mdx,json}",
-    "./components/**/*.{js,ts,jsx,tsx,mdx,json}",
-    "./app/**/*.{js,ts,jsx,tsx,mdx,json}",
-    "./src/**/*.{js,ts,jsx,tsx,mdx,json}",
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
Could you add the json files back to within the content, in the old layout it had issues reading CSS within classNames from JSON files due to this.

Reply:
Went ahead and fixed this, thank you for pointing this out!